### PR TITLE
adds basic tasks via Jake: build, test, lint and watchTask

### PR DESCRIPTION
**Pull request by ajmoorexyz**

### CODE REVIEW

File: packages/fifteen-logic/package.json [hunk 1]
```diff
@@ -1,27 +1,20 @@
 {
   "name": "fifteen-logic",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
```
Comment (sappharx) [inline packages/fifteen-logic/package.json:5]: what happened to description and main?

File: packages/fifteen-logic/Jakefile.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  desc('Run TypeScript Compiler')
+  task('build', function() {
+    exec('node_modules/.bin/tsc')
+    console.log('\n\ntsc complete')
+  })
+
+  desc('Run tests with AVA following tsc')
+  task('test', ['build'], function() {
+    exec('node_modules/.bin/ava dist/*.js')
```
Comment (sappharx) [inline packages/fifteen-logic/Jakefile.js]: it looks like we'll need to use 'dist/_.test.js' and 'dist/__/_.test.js' b/c ava is trying to run tests for every js file right now

File: packages/fifteen-logic/package.json (hunk 1 above)
Comment (ajmoorexyz) [inline packages/fifteen-logic/package.json:5]: @sappharx I added a description but main isn't being leveraged so I yanked it. Thoughts/improvements?

**Comment by ajmoorexyz** [comment]
@sappharx ava now points to *.test.js files

**Pull request merged**